valqueries-automapper/src/main/java/com/valqueries/automapper/elements/SimpleElement.java

```diff
@@ -24,7 +24,7 @@ public SimpleElement(ValqueriesQueryImpl<?> query, Property.PropertyValue<?> pro
 	}
 
 	public String queryString() {
-		return query.getTableAlias()+"."+dialect.escapeColumnOrTable(sqlNameFormatter.column(propertyValue.getProperty().getToken()))+" "+operator+" (:"+field+")";
+		return query.getTableAlias()+"."+dialect.column(propertyValue.getProperty().getToken())+" "+operator+" (:"+field+")";
```

Comment:
```suggestion
		return query.getTableAlias()+"."+dialect.column(propertyValue.getProperty())+" "+operator+" (:"+field+")";
```
so it considers DbName into it